nb/ide.branding/o.n.core/src/org/netbeans/core/Bundle_nb.properties

```diff
@@ -18,9 +18,10 @@
 # {0} - class name of exception
 # {1} - path to system folder
 NTF_ExceptionalException=\
-        A {0} exception has occurred.\n\
-	Please report this at https://netbeans.apache.org/nb/report-issue,\n\
-	including a copy of your messages.log file as an attachment.\n\
-	The messages.log file is located in your {1} folder.
+    A <code>{0}</code> has occurred.<br/><br/>\
+	If you are running the latest version of NetBeans you can \
+    <a href="https://netbeans.apache.org/nb/report-issue">report this</a>, \
+	including a copy of your messages.log file as an attachment.<br/><br/>\
+	The messages.log file is located in <a href="file://{1}">this folder</a>.
```

Comment:
@lkishalmi you think I should add a hyperlink to "latest" too with leads to the download page?

Reply:
done